metashare/api/models.py

```diff
@@ -410,6 +411,8 @@ class Task(
     currently_creating_pr = models.BooleanField(default=False)
     pr_number = models.IntegerField(null=True, blank=True)
     pr_is_open = models.BooleanField(default=False)
+    review_submitted_at = models.DateTimeField(null=True)
+    review_valid = models.BooleanField(default=False)
```

Comment:
@wlonk We'll probably want to store `review_status` also, so we can display that on the front end.

Reply:
Is that "whether this was approved, commented, or changed-requested"?

Reply:
Yes. We need to know whether it was approved or changes were requested.